sdk/turing/router/config/router_ensembler_config.py

```diff
@@ -350,6 +363,22 @@ def service_account(self) -> str:
     def service_account(self, service_account: str):
         self._service_account = service_account
 
+    @staticmethod
+    def from_config(config: turing.generated.models.EnsemblerDockerConfig):
```

Comment:
CMIIW, this method is not being overwritten anywhere else since this class is never used by another subclass, is there a reason why this is not a `classmethod` (since we're initializing the class through this call)?

Reply:
I chose to use `staticmethod` because we don't need access to the class properties. But, as you're saying, we are in fact initializing the class through the call so it makes more sense for it to be a class method. I'll update this. Thanks!